### DISCUSSION TITLE
Stability issue error throw by google play console pre-launch report details

### DISCUSSION THREAD

**Discussion by balaedit** [airsdk/Adobe-Runtime-Support#2133] [General]
Pre-launch report details throw stability issue : 

ANR in com.google.android.apps.wellbeing;PID: 5865;Broadcast of Intent { act=android.intent.action.LOCALE_CHANGED flg=0x1200810 cmp=com.google.android.apps.wellbeing/.winddown.impl.WindDownRefreshStateBroadcastReceiver_Receiver };

Full logcat : 
[logcat.txt](https://github.com/airsdk/Adobe-Runtime-Support/files/9539776/logcat.txt)

Is there any issue with my app? @ajwfrost pls suggest what I am doing wrong in my app.

**Comment by ajwfrost** [comment]
Hi

That ANR looks unrelated ..? there's a few Google apps that are being killed, whatever the "wellbeing" app it, I'm assuming that's not your one..?

Yours has some logging:
```
09-09 14:12:56.308: W/FirebaseApp(9026): Default FirebaseApp failed to initialize because no default options were found. This usually means that com.google.gms:google-services was not applied to your gradle project.
09-09 14:12:56.308: I/FirebaseInitProvider(9026): FirebaseApp initialization unsuccessful


09-09 14:18:06.623: E/AndroidRuntime(9026): FATAL EXCEPTION: Instr: androidx.test.runner.AndroidJUnitRunner
09-09 14:18:06.623: E/AndroidRuntime(9026): Process: air.EduLearningApp, PID: 9026

09-09 14:18:07.057: E/ActivityControllerCore(744): PROCESS CRASHED
09-09 14:18:07.057: E/ActivityControllerCore(744): processName: air.EduLearningApp
09-09 14:18:07.057: E/ActivityControllerCore(744): pid: 9026
09-09 14:18:07.057: E/ActivityControllerCore(744): shortMsg: android.os.DeadObjectException
09-09 14:18:07.057: E/ActivityControllerCore(744): longMsg: android.os.DeadObjectException: Error while disconnecting UiAutomation@fab7dbf[id=-1, flags=0]
09-09 14:18:07.057: E/ActivityControllerCore(744): timeMillis: 1662758286949
09-09 14:18:07.057: E/ActivityControllerCore(744): stackTrace:
09-09 14:18:07.057: E/ActivityControllerCore(744): java.lang.RuntimeException: Error while disconnecting UiAutomation@fab7dbf[id=-1, flags=0]

09-09 14:18:07.658: I/ActivityManager(611): Process air.EduLearningApp (pid 9026) has died: fg  FGS 
09-09 14:18:07.838: W/ActivityManager(611): Crash of app air.EduLearningApp running instrumentation ComponentInfo{androidx.test.tools.crawler/androidx.test.runner.AndroidJUnitRunner}
```

It basically looks like there's some test automation going on that caused a problem.. certainly there are a lot of other reports of "non-SDK API usage" which is all Google/Androidx stuff, maybe the ui automation tools need updating? Nothing that I can see about an ANR or any crashes in the AIR code though...

thanks

**Reply by balaedit** [reply]
Thank you so much. And I was waiting for the Window 32bit and 64bit combine exe from Air SDK. Is there any roadmap on this or any kind of tools which we can use it. Suggest me if possible.

**Reply by ajwfrost** [reply]
There's no plan/support for combining executables on Windows. Theoretically it may be possible to have a 32-bit application that can then re-execute itself with a 64-bit mode but for DLLs that's not possible I believe, so Windows will just stay with separate 32-bit vs 64-bit options (unless/until Microsoft change something fairly fundamental..)

32-bit windows apps work everywhere so are still the simplest option just in case someone still has a 32-bit version of the OS. We had been planning to move to 64-bit defaults for everything, including the shared AIR runtime, but it would cause problems e.g. for any already-installed AIR apps that are 32-bit and therefore can't load a 64-bit DLL, etc.

thanks
